ubuntudesign/canonical-vanilla-theme
# add strips module and update config for Canonical defaults

Done:

Added ./scss/modules/_strips.scss
Configured link colours and site max width variables
Updated markup in demo to replicate canonical.com's usage (.inner-wrappers in rows, etc.)

QA:
 - Checkout git@github.com:ubuntudesign/vanilla-framework.git into node_modules
 - gulp build 
 - In-browser check of ./demo/index.html (check for correct display of the new rows, link colours (ubuntu orange) and that the site max width is 984px)
